### DISCUSSION TITLE
Slack Channel Restricted

### DISCUSSION THREAD

**Discussion by addy-zero** [dropbox/Store#408] [General]
The [Slack Channel](http://kotlin.slack.com/) provided on the Dropbox GSoC page does not seem to allow for registration.

What can be done in this case, and are there any other modes of communication for interested candidates to understand the projects better?

Edit: The provided email is also non-functional. Please give an update.

**Comment by Atharvat** [comment]
Yes, I am also facing the same problem. The slack channel is restricted and the email is invalid. Please provide an update.

**Comment by shankarpriyank** [comment]
Hey @digitalbuddha @chris-mitchell @rharter can you guys help?

**Comment by nehalsep7** [comment]
What is the slack channel? Can someone add me? Tried the one in description and it doesn't work

**Comment by matt-ramotar** [comment]
Hey all - We created a Slack channel in the Android Dev Social workspace
- [Workspace invitation](https://join.slack.com/t/androiddev-social/shared_invite/zt-1ndtvrsvk-O7dsgI_fzXipVpWkXVIgdQ) 
- [Channel link](https://androiddev-social.slack.com/archives/C04JHS67B7C)

**Comment by matt-ramotar** [comment]
Store now has an official channel in kotlinlang! https://kotlinlang.slack.com/archives/C06007Z01HU
